mobile_robot_odometry:
  leftwheel_linkname: MobileRobot::wheel_1
  rightwheel_linkname: MobileRobot::wheel_3
  base_link_id : "/custom_base"
  odom_link_id : "/custom_odom"

